### DISCUSSION TITLE
【新功能建议】为翻译后的中文内容增加下划线、方框或高亮的功能

### DISCUSSION THREAD

**Discussion by night-cruise** [immersive-translate/immersive-translate#426] [Ideas]
### 新功能 | New Feature

1. 为翻译后的中文内容增加下划线或方框的功能

我正在使用的一款划词翻译插件，在翻译后会为中文内容添加下划线或方框，就像这样：
![屏幕截图 2023-05-04 004058](https://user-images.githubusercontent.com/77157236/235984326-669bf20b-3bc0-4d94-bb3a-f24506a465b9.png)

或者像这样：
![屏幕截图 2023-05-04 004853](https://user-images.githubusercontent.com/77157236/235984830-1a40a495-b698-41b7-8d46-946ac982a9f0.png)

可以在翻译插件中添加一个选项，让用户自己选择是否设置方框、下划线以及它们的颜色之类的。

### 为什么您想要添加这些功能 | Why you want to add these features

1. 我认为使用双语翻译插件的很多用户主要还是想要专注于翻译后的中文内容，只有在感觉翻译不通顺的时候才会去对照原来的英文单词。像这样翻译前后的英文和中文内容样式完全一致的话，感觉阅读起来视觉就比较容易疲劳，很容易被英文内容干扰：

![屏幕截图 2023-05-04 004853](https://user-images.githubusercontent.com/77157236/235987300-660393a0-440c-410e-bb39-1707ecaa98f2.png)

### 提议 | Proposal

虽然我也是计算机专业的学生，但是我完全不会 typescript，所以也做不了贡献😂

**Comment by theowenyoung** [comment]
已有该功能，在界面设置 中选择译文样式即可～

**Comment by night-cruise** [comment]
> 已有该功能，在界面设置 中选择译文样式即可～

OK，非常感谢您的回答。
